### PR TITLE
refactor getChapter

### PR DESCRIPTION
**Pull request by AriaMoradi**
I tried practicing the third chapter of Clean Code by Robert C. Martin

Same concept in video: https://youtu.be/QvrNwfuWbCQ

### CODE REVIEW

File: server/src/main/kotlin/suwayomi/tachidesk/manga/impl/chapter/ChapterForDownload.kt [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        val chapterId = chapterEntry[ChapterTable.id].value
+
+        val chapterDir = getChapterDir(mangaId, chapterId)
+
+        println(chapterDir)
+        println(getPageName(0))
+
+        return ImageResponse.findFileNameStartingWith(
+            chapterDir,
+            getPageName(0)
```
Comment (Syer10) [inline server/src/main/kotlin/suwayomi/tachidesk/manga/impl/chapter/ChapterForDownload.kt:145]:
```suggestion
            getPageName(1)
```

File: server/src/main/kotlin/suwayomi/tachidesk/manga/impl/chapter/ChapterForDownload.kt [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        return !(chapterEntry[ChapterTable.isDownloaded] && firstPageExists())
+    }
+
+    private fun firstPageExists(): Boolean {
+        val chapterId = chapterEntry[ChapterTable.id].value
+
+        val chapterDir = getChapterDir(mangaId, chapterId)
+
+        println(chapterDir)
+        println(getPageName(0))
```
Comment (Syer10) [inline server/src/main/kotlin/suwayomi/tachidesk/manga/impl/chapter/ChapterForDownload.kt:141]:
```suggestion
        println(getPageName(1))
```

**Review by Syer10 — CHANGES_REQUESTED** [review]
Looks good otherwise

File: server/src/main/kotlin/suwayomi/tachidesk/manga/impl/chapter/ChapterForDownload.kt (hunk 1 above)
Comment (AriaMoradi) [reply server/src/main/kotlin/suwayomi/tachidesk/manga/impl/chapter/ChapterForDownload.kt:145]: that's was a bug

It was checking for the second page

**Review by Syer10 — APPROVED** [review]

**Pull request merged**